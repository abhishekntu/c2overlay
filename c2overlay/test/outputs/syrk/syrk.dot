digraph graphname {
N55 [color="black", label="sqr_N49_N55"];
N74 [color="black", label="store_Imm_4_N94_N74"];
N70 [color="black", label="store_Imm_1_N93_N70"];
N54 [color="black", label="N79N20_mul_Imm_9_N79_add_N20_N54"];
N73 [color="black", label="store_Imm_5_N97_N73"];
N69 [color="black", label="store_Imm_7_N92_N69"];
N58 [color="black", label="mul_Imm_10_N40_N58"];
N68 [color="black", label="store_Imm_6_N99_N68"];
N49 [color="black", label="N60N87_mul_N60_add_N87_N49"];
N48 [color="black", label="mul_Imm_10_N61_N48"];
N65 [color="black", label="mul_N50_N65"];
N64 [color="black", label="N52N71_mul_N52_add_N71_N64"];
N63 [color="black", label="mul_N59_N63"];
N62 [color="black", label="N58N21_mul_N58_add_N21_N62"];
N61 [color="black", label="mul_Imm_10_N46_N61"];
N60 [color="black", label="N45N57_mul_Imm_9_N45_add_N57_N60"];
N41 [color="black", label="sqr_N90_N41"];
N40 [color="black", label="mul_Imm_10_N37_N40"];
N43 [color="black", label="N66N68_mul_N66_add_N68_N43"];
N42 [color="black", label="sqr_N67_N42"];
N45 [color="black", label="sqr_N64_N45"];
N44 [color="black", label="sqr_N65_N44"];
N47 [color="black", label="N62N80_mul_Imm_9_N62_add_N80_N47"];
N46 [color="black", label="N63N54_add_N63_add_N54_N46"];
N29 [color="black", label="N82N53_mul_Imm_9_N82_add_N53_N29"];
N28 [color="black", label="N83N56_mul_N83_add_N56_N28"];
N59 [color="black", label="mul_Imm_10_N42_N59"];
N23 [color="black", label="N27N77_mul_Imm_9_N27_add_N77_N23"];
N22 [color="black", label="N22N72_mul_N22_add_N72_N22"];
N21 [color="black", label="sqr_N23_N21"];
N20 [color="black", label="N28N75_mul_N28_add_N75_N20"];
N27 [color="black", label="mul_Imm_10_N86_N27"];
N26 [color="black", label="N85N44_mul_Imm_9_N85_add_N44_N26"];
N25 [color="black", label="N25N74_mul_N25_add_N74_N25"];
N24 [color="black", label="N26N24_add_N26_add_N24_N24"];
N67 [color="black", label="store_Imm_3_N98_N67"];
N75 [color="black", label="store_Imm_8_N95_N75"];
N66 [color="black", label="N51N73_mul_Imm_9_N51_add_N73_N66"];
N8 [color="black", label="load_Imm_3_N15_N8"];
N9 [color="black", label="load_Imm_1_N18_N9"];
N71 [color="black", label="store_Imm_2_N91_N71"];
N1 [color="black", label="load_Imm_0_N12_N1"];
N2 [color="black", label="load_Imm_5_N13_N2"];
N3 [color="black", label="load_Imm_7_N10_N3"];
N4 [color="black", label="load_Imm_2_N11_N4"];
N5 [color="black", label="load_Imm_1_N16_N5"];
N6 [color="black", label="load_Imm_8_N17_N6"];
N7 [color="black", label="load_Imm_0_N14_N7"];
N12 [color="black", label="load_Imm_7_N1_N12"];
N13 [color="black", label="load_Imm_2_N2_N13"];
N10 [color="black", label="load_Imm_8_N8_N10"];
N11 [color="black", label="load_Imm_6_N9_N11"];
N16 [color="black", label="load_Imm_3_N5_N16"];
N17 [color="black", label="load_Imm_4_N6_N17"];
N14 [color="black", label="load_Imm_4_N3_N14"];
N15 [color="black", label="load_Imm_6_N4_N15"];
N56 [color="black", label="sqr_N48_N56"];
N57 [color="black", label="N41N43_add_N41_add_N43_N57"];
N18 [color="black", label="load_Imm_5_N7_N18"];
N19 [color="black", label="N29N47_mul_Imm_9_N29_add_N47_N19"];
N52 [color="black", label="mul_N70_N52"];
N53 [color="black", label="N78N81_mul_N78_add_N81_N53"];
N50 [color="black", label="N69N88_mul_Imm_9_N69_add_N88_N50"];
N51 [color="black", label="mul_N76_N51"];
N38 [color="black", label="sqr_N35_N38"];
N39 [color="black", label="N36N19_mul_N36_add_N19_N39"];
N72 [color="black", label="store_Imm_0_N96_N72"];
N30 [color="black", label="sqr_N89_N30"];
N31 [color="black", label="N38N84_mul_N38_add_N84_N31"];
N32 [color="black", label="mul_N39_N32"];
N33 [color="black", label="N30N55_mul_N30_add_N55_N33"];
N34 [color="black", label="mul_Imm_10_N31_N34"];
N35 [color="black", label="mul_N32_N35"];
N36 [color="black", label="mul_Imm_10_N33_N36"];
N37 [color="black", label="mul_Imm_10_N34_N37"];
N55 -> N57;
N54 -> N67;
N58 -> N54;
N49 -> N50;
N48 -> N66;
N65 -> N22;
N64 -> N66;
N63 -> N62;
N62 -> N49;
N61 -> N26;
N60 -> N73;
N41 -> N24;
N40 -> N19;
N43 -> N53;
N42 -> N57;
N45 -> N46;
N44 -> N46;
N47 -> N70;
N46 -> N29;
N29 -> N75;
N28 -> N60;
N59 -> N23;
N23 -> N69;
N22 -> N64;
N21 -> N24;
N20 -> N23;
N27 -> N47;
N26 -> N72;
N25 -> N20;
N24 -> N19;
N66 -> N71;
N8 -> N58;
N9 -> N27;
N1 -> N51;
N1 -> N22;
N1 -> N63;
N1 -> N31;
N1 -> N56;
N2 -> N39;
N2 -> N41;
N2 -> N53;
N2 -> N28;
N2 -> N20;
N3 -> N62;
N3 -> N65;
N3 -> N44;
N3 -> N35;
N3 -> N25;
N4 -> N48;
N5 -> N55;
N5 -> N32;
N5 -> N62;
N5 -> N65;
N5 -> N43;
N6 -> N20;
N6 -> N49;
N6 -> N28;
N6 -> N64;
N6 -> N38;
N7 -> N61;
N12 -> N59;
N13 -> N64;
N13 -> N49;
N13 -> N42;
N13 -> N53;
N13 -> N39;
N10 -> N37;
N11 -> N45;
N11 -> N63;
N11 -> N33;
N11 -> N22;
N11 -> N52;
N16 -> N30;
N16 -> N52;
N16 -> N51;
N16 -> N31;
N16 -> N33;
N17 -> N25;
N17 -> N32;
N17 -> N43;
N17 -> N35;
N17 -> N21;
N14 -> N40;
N15 -> N36;
N56 -> N57;
N57 -> N26;
N18 -> N34;
N19 -> N74;
N52 -> N25;
N53 -> N47;
N50 -> N68;
N51 -> N43;
N38 -> N46;
N39 -> N54;
N30 -> N24;
N31 -> N39;
N32 -> N31;
N33 -> N28;
N34 -> N60;
N35 -> N33;
N36 -> N50;
N37 -> N29;
}
