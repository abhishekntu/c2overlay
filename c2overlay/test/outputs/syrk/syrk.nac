procedure foo()
{
  localvar s32 D_2409;
  localvar s32 D_2410;
  localvar s32 D_2411;
  localvar s32 D_2412;
  localvar s32 D_2413;
  localvar s32 D_2414;
  localvar s32 D_2415;
  localvar s32 D_2416;
  localvar s32 D_2417;
  localvar s32 D_2418;
  localvar s32 D_2419;
  localvar s32 D_2420;
  localvar s32 D_2421;
  localvar s32 D_2422;
  localvar s32 D_2423;
  localvar s32 D_2424;
  localvar s32 D_2425;
  localvar s32 D_2426;
  localvar s32 D_2427;
  localvar s32 D_2428;
  localvar s32 D_2429;
  localvar s32 D_2430;
  localvar s32 D_2431;
  localvar s32 D_2432;
  localvar s32 D_2433;
  localvar s32 D_2434;
  localvar s32 D_2435;
  localvar s32 D_2436;
  localvar s32 D_2437;
  localvar s32 D_2438;
  localvar s32 D_2439;
  localvar s32 D_2440;
  localvar s32 D_2441;
  localvar s32 D_2442;
  localvar s32 D_2443;
  localvar s32 D_2444;
  localvar s32 D_2445;
  localvar s32 D_2446;
  localvar s32 D_2447;
  localvar s32 D_2448;
  localvar s32 D_2449;
  localvar s32 D_2450;
  localvar s32 D_2451;
  localvar s32 D_2452;
  localvar s32 D_2453;
  localvar s32 D_2454;
  localvar s32 D_2455;
  localvar s32 D_2456;
  localvar s32 D_2457;
  localvar s32 D_2458;
  localvar s32 D_2459;
  localvar s32 D_2460;
  localvar s32 D_2461;
  localvar s32 D_2462;
  localvar s32 D_2463;
  localvar s32 D_2464;
  localvar s32 D_2465;
  localvar s32 D_2466;
  localvar s32 D_2467;
  localvar s32 D_2468;
  localvar s32 D_2469;
  localvar s32 D_2470;
  localvar s32 D_2471;
  localvar s32 D_2472;
  localvar s32 D_2473;
  localvar s32 D_2474;
  localvar s32 D_2475;
  localvar s32 D_2476;
  localvar s32 D_2477;
  localvar s32 D_2478;
  localvar s32 D_2479;
  localvar s32 D_2480;
  localvar s32 A[9];
  localvar s32 C[9];
  localvar s32 D[9];
  localvar s32 alpha;
  localvar s32 beta;
  localvar s32 i;
  localvar s32 j;
  localvar s32 t00000;
  localvar s32 t00001;
  localvar s32 t00002;
  localvar s32 t00003;
  localvar s32 t00004;
  localvar s32 t00005;
  localvar s32 t00006;
  localvar s32 t00007;
  localvar s32 t00008;
  localvar s32 t00009;
  localvar s32 t00010;
  localvar s32 t00011;
  localvar s32 t00012;
  localvar s32 t00013;
  localvar s32 t00014;
  localvar s32 t00015;
  localvar s32 t00016;
  localvar s32 t00017;
  localvar s32 t00018;
  localvar s32 t00019;
  localvar s32 t00020;
  localvar s32 t00021;
  localvar s32 t00022;
  localvar s32 t00023;
  localvar s32 t00024;
  localvar s32 t00025;
  localvar s32 t00026;
  localvar s32 t00027;
  localvar s32 t00028;
  localvar s32 t00029;
  localvar s32 t00030;
  localvar s32 t00031;
  localvar s32 t00032;
  localvar s32 t00033;
  localvar s32 t00034;
  localvar s32 t00035;
  localvar s32 t00036;
  localvar s32 t00037;
  localvar s32 t00038;
  localvar s32 t00039;
  localvar s32 t00040;
  localvar s32 t00041;
  localvar s32 t00042;
  localvar s32 t00043;
  localvar s32 t00044;
  localvar s32 t00045;
  localvar s32 t00046;
  localvar s32 t00047;
  localvar s32 t00048;
  localvar s32 t00049;
  localvar s32 t00050;
  localvar s32 t00051;
  localvar s32 t00052;
  localvar s32 t00053;
  localvar s32 t00054;
  localvar s32 t00055;
  localvar s32 t00056;
  localvar s32 t00057;
  localvar s32 t00058;
  localvar s32 t00059;
  localvar s32 t00060;
  localvar s32 t00061;
  localvar s32 t00062;
  localvar s32 t00063;
  localvar s32 t00064;
  localvar s32 t00065;
  localvar s32 t00066;
  localvar s32 t00067;
  localvar s32 t00068;
  localvar s32 t00069;
  localvar s32 t00070;
  localvar s32 t00071;
L0005:
  alpha <= ldc 9;
  beta <= ldc 10;
  D_2409 <= load C,0;
  D_2410 <= mul D_2409,beta;
  D_2411 <= load A,0;
  D_2411 <= mov D_2411;
  D_2412 <= sqr D_2411;
  D_2413 <= load A,1;
  D_2413 <= mov D_2413;
  D_2414 <= sqr D_2413;
  D_2415 <= add D_2412,D_2414;
  D_2416 <= load A,2;
  D_2416 <= mov D_2416;
  D_2417 <= sqr D_2416;
  D_2418 <= add D_2415,D_2417;
  D_2419 <= mul D_2418,alpha;
  D_2420 <= add D_2410,D_2419;
  D <= store D_2420,0;
  D_2421 <= load C,1;
  D_2422 <= mul D_2421,beta;
  D_2411 <= load A,0;
  D_2423 <= load A,3;
  D_2424 <= mul D_2411,D_2423;
  D_2413 <= load A,1;
  D_2425 <= load A,4;
  D_2426 <= mul D_2413,D_2425;
  D_2427 <= add D_2424,D_2426;
  D_2416 <= load A,2;
  D_2428 <= load A,5;
  D_2429 <= mul D_2416,D_2428;
  D_2430 <= add D_2427,D_2429;
  D_2431 <= mul D_2430,alpha;
  D_2432 <= add D_2422,D_2431;
  D <= store D_2432,1;
  D_2433 <= load C,2;
  D_2434 <= mul D_2433,beta;
  D_2411 <= load A,0;
  D_2435 <= load A,6;
  D_2436 <= mul D_2411,D_2435;
  D_2413 <= load A,1;
  D_2437 <= load A,7;
  D_2438 <= mul D_2413,D_2437;
  D_2439 <= add D_2436,D_2438;
  D_2416 <= load A,2;
  D_2440 <= load A,8;
  D_2441 <= mul D_2416,D_2440;
  D_2442 <= add D_2439,D_2441;
  D_2443 <= mul D_2442,alpha;
  D_2444 <= add D_2434,D_2443;
  D <= store D_2444,2;
  D_2445 <= load C,3;
  D_2446 <= mul D_2445,beta;
  D_2423 <= load A,3;
  D_2411 <= load A,0;
  D_2424 <= mul D_2423,D_2411;
  D_2425 <= load A,4;
  D_2413 <= load A,1;
  D_2426 <= mul D_2425,D_2413;
  D_2427 <= add D_2424,D_2426;
  D_2428 <= load A,5;
  D_2416 <= load A,2;
  D_2429 <= mul D_2428,D_2416;
  D_2430 <= add D_2427,D_2429;
  D_2431 <= mul D_2430,alpha;
  D_2447 <= add D_2446,D_2431;
  D <= store D_2447,3;
  D_2448 <= load C,4;
  D_2449 <= mul D_2448,beta;
  D_2423 <= load A,3;
  D_2423 <= mov D_2423;
  D_2450 <= sqr D_2423;
  D_2425 <= load A,4;
  D_2425 <= mov D_2425;
  D_2451 <= sqr D_2425;
  D_2452 <= add D_2450,D_2451;
  D_2428 <= load A,5;
  D_2428 <= mov D_2428;
  D_2453 <= sqr D_2428;
  D_2454 <= add D_2452,D_2453;
  D_2455 <= mul D_2454,alpha;
  D_2456 <= add D_2449,D_2455;
  D <= store D_2456,4;
  D_2457 <= load C,5;
  D_2458 <= mul D_2457,beta;
  D_2423 <= load A,3;
  D_2435 <= load A,6;
  D_2459 <= mul D_2423,D_2435;
  D_2425 <= load A,4;
  D_2437 <= load A,7;
  D_2460 <= mul D_2425,D_2437;
  D_2461 <= add D_2459,D_2460;
  D_2428 <= load A,5;
  D_2440 <= load A,8;
  D_2462 <= mul D_2428,D_2440;
  D_2463 <= add D_2461,D_2462;
  D_2464 <= mul D_2463,alpha;
  D_2465 <= add D_2458,D_2464;
  D <= store D_2465,5;
  D_2466 <= load C,6;
  D_2467 <= mul D_2466,beta;
  D_2435 <= load A,6;
  D_2411 <= load A,0;
  D_2436 <= mul D_2435,D_2411;
  D_2437 <= load A,7;
  D_2413 <= load A,1;
  D_2438 <= mul D_2437,D_2413;
  D_2439 <= add D_2436,D_2438;
  D_2440 <= load A,8;
  D_2416 <= load A,2;
  D_2441 <= mul D_2440,D_2416;
  D_2442 <= add D_2439,D_2441;
  D_2443 <= mul D_2442,alpha;
  D_2468 <= add D_2467,D_2443;
  D <= store D_2468,6;
  D_2469 <= load C,7;
  D_2470 <= mul D_2469,beta;
  D_2435 <= load A,6;
  D_2423 <= load A,3;
  D_2459 <= mul D_2435,D_2423;
  D_2437 <= load A,7;
  D_2425 <= load A,4;
  D_2460 <= mul D_2437,D_2425;
  D_2461 <= add D_2459,D_2460;
  D_2440 <= load A,8;
  D_2428 <= load A,5;
  D_2462 <= mul D_2440,D_2428;
  D_2463 <= add D_2461,D_2462;
  D_2464 <= mul D_2463,alpha;
  D_2471 <= add D_2470,D_2464;
  D <= store D_2471,7;
  D_2472 <= load C,8;
  D_2473 <= mul D_2472,beta;
  D_2435 <= load A,6;
  D_2435 <= mov D_2435;
  D_2474 <= sqr D_2435;
  D_2437 <= load A,7;
  D_2437 <= mov D_2437;
  D_2475 <= sqr D_2437;
  D_2476 <= add D_2474,D_2475;
  D_2440 <= load A,8;
  D_2440 <= mov D_2440;
  D_2477 <= sqr D_2440;
  D_2478 <= add D_2476,D_2477;
  D_2479 <= mul D_2478,alpha;
  D_2480 <= add D_2473,D_2479;
  D <= store D_2480,8;
  L_EXIT <= jmpun;
L_EXIT:
  nop;
}
