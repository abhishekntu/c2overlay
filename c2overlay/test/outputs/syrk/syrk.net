.input N8
pinlist: N8

.input N9
pinlist: N9

.input N1
pinlist: N1

.input N2
pinlist: N2

.input N3
pinlist: N3

.input N4
pinlist: N4

.input N5
pinlist: N5

.input N6
pinlist: N6

.input N7
pinlist: N7

.input N12
pinlist: N12

.input N13
pinlist: N13

.input N10
pinlist: N10

.input N11
pinlist: N11

.input N16
pinlist: N16

.input N17
pinlist: N17

.input N14
pinlist: N14

.input N15
pinlist: N15

.input N18
pinlist: N18

.output out:N70
pinlist: N70

.output out:N69
pinlist: N69

.output out:N68
pinlist: N68

.output out:N65
pinlist: N65

.output out:N64
pinlist: N64

.output out:N67
pinlist: N67

.output out:N66
pinlist: N66

.output out:N71
pinlist: N71

.output out:N72
pinlist: N72

.clb N55_blk
pinlist: N41 N52 N57 open N69 open open open open 
subblock: N55_blk 0 1 2 open 4 open open open open

.clb N54_blk
pinlist: N8 open open open N54 open open open open 
subblock: N54_blk 0 open open open 4 open open open open

.clb N58_blk
pinlist: N3 N5 N59 open N58 open open open open 
subblock: N58_blk 0 1 2 open 4 open open open open

.clb N49_blk
pinlist: N11 N16 open open N49 open open open open 
subblock: N49_blk 0 1 open open 4 open open open open

.clb N48_blk
pinlist: N1 N16 open open N48 open open open open 
subblock: N48_blk 0 1 open open 4 open open open open

.clb N63_blk
pinlist: N3 N5 open open N63 open open open open 
subblock: N63_blk 0 1 open open 4 open open open open

.clb N62_blk
pinlist: N6 N13 open open N62 open open open open 
subblock: N62_blk 0 1 open open 4 open open open open

.clb N61_blk
pinlist: N35 N36 N43 open N72 open open open open 
subblock: N61_blk 0 1 2 open 4 open open open open

.clb N60_blk
pinlist: N27 N31 N32 open N70 open open open open 
subblock: N60_blk 0 1 2 open 4 open open open open

.clb N41_blk
pinlist: N13 open open open N41 open open open open 
subblock: N41_blk 0 open open open 4 open open open open

.clb N40_blk
pinlist: N29 N37 N54 open N64 open open open open 
subblock: N40_blk 0 1 2 open 4 open open open open

.clb N43_blk
pinlist: N3 N44 open open N43 open open open open 
subblock: N43_blk 0 1 open open 4 open open open open

.clb N42_blk
pinlist: N5 N17 N48 open N42 open open open open 
subblock: N42_blk 0 1 2 open 4 open open open open

.clb N45_blk
pinlist: N4 open open open N45 open open open open 
subblock: N45_blk 0 open open open 4 open open open open

.clb N44_blk
pinlist: N11 open open open N44 open open open open 
subblock: N44_blk 0 open open open 4 open open open open

.clb N47_blk
pinlist: N19 N22 N56 open N66 open open open open 
subblock: N47_blk 0 1 2 open 4 open open open open

.clb N46_blk
pinlist: N6 N13 open open N46 open open open open 
subblock: N46_blk 0 1 open open 4 open open open open

.clb N29_blk
pinlist: N30 N1 N16 open N29 open open open open 
subblock: N29_blk 0 1 2 open 4 open open open open

.clb N28_blk
pinlist: N16 open open open N28 open open open open 
subblock: N28_blk 0 open open open 4 open open open open

.clb N59_blk
pinlist: N1 N11 open open N59 open open open open 
subblock: N59_blk 0 1 open open 4 open open open open

.clb N23_blk
pinlist: N20 N38 N39 open N71 open open open open 
subblock: N23_blk 0 1 2 open 4 open open open open

.clb N22_blk
pinlist: N3 N17 N49 open N22 open open open open 
subblock: N22_blk 0 1 2 open 4 open open open open

.clb N21_blk
pinlist: N1 N11 N63 open N21 open open open open 
subblock: N21_blk 0 1 2 open 4 open open open open

.clb N20_blk
pinlist: N28 N17 open open N20 open open open open 
subblock: N20_blk 0 1 open open 4 open open open open

.clb N27_blk
pinlist: N2 N6 open open N27 open open open open 
subblock: N27_blk 0 1 open open 4 open open open open

.clb N26_blk
pinlist: N25 N42 N51 open N67 open open open open 
subblock: N26_blk 0 1 2 open 4 open open open open

.clb N25_blk
pinlist: N9 open open open N25 open open open open 
subblock: N25_blk 0 open open open 4 open open open open

.clb N24_blk
pinlist: N34 N46 N58 open N65 open open open open 
subblock: N24_blk 0 1 2 open 4 open open open open

.clb N56_blk
pinlist: N12 open open open N56 open open open open 
subblock: N56_blk 0 open open open 4 open open open open

.clb N57_blk
pinlist: N7 open open open N57 open open open open 
subblock: N57_blk 0 open open open 4 open open open open

.clb N19_blk
pinlist: N2 N6 open open N19 open open open open 
subblock: N19_blk 0 1 open open 4 open open open open

.clb N52_blk
pinlist: N5 N53 open open N52 open open open open 
subblock: N52_blk 0 1 open open 4 open open open open

.clb N53_blk
pinlist: N1 open open open N53 open open open open 
subblock: N53_blk 0 open open open 4 open open open open

.clb N50_blk
pinlist: N21 N45 N62 open N68 open open open open 
subblock: N50_blk 0 1 2 open 4 open open open open

.clb N51_blk
pinlist: N2 N13 open open N51 open open open open 
subblock: N51_blk 0 1 open open 4 open open open open

.clb N38_blk
pinlist: N14 open open open N38 open open open open 
subblock: N38_blk 0 open open open 4 open open open open

.clb N39_blk
pinlist: N2 open open open N39 open open open open 
subblock: N39_blk 0 open open open 4 open open open open

.clb N30_blk
pinlist: N5 N17 open open N30 open open open open 
subblock: N30_blk 0 1 open open 4 open open open open

.clb N31_blk
pinlist: N33 N11 N16 open N31 open open open open 
subblock: N31_blk 0 1 2 open 4 open open open open

.clb N32_blk
pinlist: N18 open open open N32 open open open open 
subblock: N32_blk 0 open open open 4 open open open open

.clb N33_blk
pinlist: N3 N17 open open N33 open open open open 
subblock: N33_blk 0 1 open open 4 open open open open

.clb N34_blk
pinlist: N15 open open open N34 open open open open 
subblock: N34_blk 0 open open open 4 open open open open

.clb N35_blk
pinlist: N10 open open open N35 open open open open 
subblock: N35_blk 0 open open open 4 open open open open

.clb N36_blk
pinlist: N6 open open open N36 open open open open 
subblock: N36_blk 0 open open open 4 open open open open

.clb N37_blk
pinlist: N2 N13 open open N37 open open open open 
subblock: N37_blk 0 1 open open 4 open open open open

