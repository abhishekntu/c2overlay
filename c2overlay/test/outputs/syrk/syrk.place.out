Netlist file: test.net   Architecture file: test.arch.xml
Array size: 7 x 7 logic blocks

#block name	x	y	subblk	block number
#----------	--	--	------	------------
N8		8	7	0	#0
N9		4	8	0	#1
N1		2	0	0	#2
N2		8	5	0	#3
N3		0	3	0	#4
N4		0	4	0	#5
N5		5	0	0	#6
N6		0	6	0	#7
N7		3	0	0	#8
N12		7	8	0	#9
N13		8	4	0	#10
N10		1	0	0	#11
N11		0	2	0	#12
N16		6	0	0	#13
N17		8	3	0	#14
N14		8	1	0	#15
N15		1	8	0	#16
N18		3	8	0	#17
out:N74		8	2	0	#18
out:N70		5	8	0	#19
out:N73		2	8	0	#20
out:N69		8	6	0	#21
out:N68		0	7	0	#22
out:N67		6	8	0	#23
out:N75		0	1	0	#24
out:N71		0	5	0	#25
out:N72		4	0	0	#26
N55_blk		6	1	0	#27
N54_blk		6	7	0	#28
N58_blk		7	7	0	#29
N49_blk		3	5	0	#30
N48_blk		1	4	0	#31
N65_blk		2	3	0	#32
N64_blk		2	5	0	#33
N63_blk		3	2	0	#34
N62_blk		3	3	0	#35
N61_blk		3	1	0	#36
N60_blk		2	6	0	#37
N41_blk		7	5	0	#38
N40_blk		7	1	0	#39
N43_blk		5	3	0	#40
N42_blk		5	4	0	#41
N45_blk		2	2	0	#42
N44_blk		1	3	0	#43
N47_blk		5	6	0	#44
N46_blk		1	2	0	#45
N29_blk		1	1	0	#46
N28_blk		3	6	0	#47
N59_blk		7	6	0	#48
N23_blk		6	6	0	#49
N22_blk		2	4	0	#50
N21_blk		7	4	0	#51
N20_blk		4	6	0	#52
N27_blk		5	7	0	#53
N26_blk		4	1	0	#54
N25_blk		4	5	0	#55
N24_blk		7	3	0	#56
N66_blk		1	5	0	#57
N56_blk		5	2	0	#58
N57_blk		5	1	0	#59
N19_blk		7	2	0	#60
N52_blk		4	2	0	#61
N53_blk		5	5	0	#62
N50_blk		2	7	0	#63
N51_blk		4	3	0	#64
N38_blk		1	6	0	#65
N39_blk		6	5	0	#66
N30_blk		6	2	0	#67
N31_blk		6	4	0	#68
N32_blk		6	3	0	#69
N33_blk		3	4	0	#70
N34_blk		3	7	0	#71
N35_blk		4	4	0	#72
N36_blk		1	7	0	#73
N37_blk		2	1	0	#74
